src/components/mobile-docs-sidebar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import Sidebar, { SidebarProps } from 'components/sidebar'
+import { useState } from 'react'
+
+interface MobileDocsSidebar {
+  levels: SidebarProps[]
```

Comment:
💭  We could potentially include some of the `levelButtonProps` attributes into this interface, so each level can declare how its level button can render

Reply:
What are some things you're thinking of that should be declarable by level? I think the text can be customizable, but I think we can accomplish that if maybe we rename `backToLinkProps` to something like `backToProps` since the text is the same regardless of whether it's a `Button` or `StandaloneLink` getting rendered. Things like `iconPosition` and `onClick` highly depend on the number of levels, which `MobileDocsSidebar` manages so the levels aren't needing to be aware of any other levels around them.

Reply:
The `levelButtonProps` are declared by level below, but yeah having those managed by `MobileDocsSidebar` seems reasonable if the navigation between levels is linear (e.g. level navigation is always sequential). Agreed that `backToLinksProps` would be an appropriate place for any customization we need to make to the link/button! 👍

Reply:
Removed this file and component, so resolving this thread! ✨